doc/_includes/DynamicIts.hs

```diff
@@ -0,0 +1,14 @@
+import Test.Hspec
+import Control.Monad
+import System.IO
+import System.Directory
+
+main :: IO ()
+main = hspec $ do
+  describe "generate specs" $ do
+    allFiles <- runIO (getDirectoryContents ".")
+    let files = filter (`notElem` [".", ".."]) allFiles
+    forM_ files $ \file -> do
+      it ("test using the file " ++ file) $ do
+        pending
+
```

Comment:
This example is still not executed nor included into the documentation. To make that happen you would need to include it somewhere, in a very similar way to what is done for the other examples.